packages/vscode-extension/README.md

```diff
@@ -1,18 +1,17 @@
 # Microsoft Teams Toolkit for Visual Studio Code
-
-## What is the Teams Toolkit?
+## What is  Teams Toolkit?
```

Comment:
minor: double spaces here, but it looks good when rendered as Markdown.

Reply:
fixed